jenkins/jenkins.groovy

```diff
@@ -0,0 +1,52 @@
+//-------------------------------------------------------------------
+// File: jenkins.groovy
+// Description: A util file for hnsesb jenkins jobs
+//-------------------------------------------------------------------
+//This method returns namespace name with license plate prefix
+def project(val) {
+	return "c5839f-" + val.toLowerCase()
+}
+// Convert value to lowercase
+def lower(val) {
+	return val.toLowerCase()
+}
+// Prefix hnsesb- with passed value. This method is used for defining application name
```

Comment:
Comment is opposite of code, should it be "passed value is prefixed with hnesb-".

Reply:
Updated